zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2466,6 +2466,72 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /users/{userKey}/authorizations/search:
+    post:
+      tags:
+        - Authorization
+      summary: "Query a user's authorizations (alpha)"
```

🔧 To be consistent, this should just be "Query user authorizations (alpha)"
```suggestion
      summary: "Query user authorizations (alpha)"
```